packages/ng/src/app/user/picture/user-picture.component.ts

```diff
@@ -38,7 +48,7 @@ export class LuUserPictureComponent {
 
 	private getNameHue() {
 		// we sum the chars in user's firstname + lastname
-		const charSum = this.displayPipe.transform(this._user, 'lf')
+		const charSum = this.displayPipe.transform(this._user, 'fl')

```

Comment:
?

Reply:
it changes nothing cuz its still a sum of all charcodes of firstname + lastname

Reply:
yes I know, it's just for code consistency as fl is our default everywhere.